test/e2e/specs/analyticsPageVideoIsPlaying.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { vpTest } from '../fixtures/vpTest';
```

Comment:
change the file name to analyticsPage.spec.ts, wite here all the tests for that page

Reply:
But currently we're not sure if we'll have additional tests for this page.